doc/psidk/src/development/services/rust-service/package.md

```diff
@@ -23,9 +23,13 @@ version = "0.1.0"
 [package.metadata.psibase]
```

Comment:
Still doesn't show use of the `services` field, but perhaps this is better left documented in a future development guide.

Reply:
`services` is used to make the package crate separate from any of the services that compose it. I thought that having an example with 3 different Cargo.toml files would be more distracting than useful.